src/app/Dashboard/components/DashboardComponent.js

```diff
@@ -107,5 +142,6 @@ export default connect(
   mapStateToProps,
   {
     fetchUserActivites: actions.fetchUserActivitiesRequest,
+    loadCategories: hactions.loadCategories,
```

Comment:
what is `hactions`?

Reply:
@kimani-ndegwa I have removed the hactions file and transferred all its contents which loadCategories action to the actions file. I initially had two files containing actions but now they have been consolidated to one.